src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c

```diff
@@ -620,19 +624,18 @@ static void copy_statx_attributes(JNIEnv* env, struct my_statx* buf, jobject att
     (*env)->SetLongField(env, attrs, attrs_st_atime_sec, (jlong)buf->stx_atime.tv_sec);
     (*env)->SetLongField(env, attrs, attrs_st_mtime_sec, (jlong)buf->stx_mtime.tv_sec);
     (*env)->SetLongField(env, attrs, attrs_st_ctime_sec, (jlong)buf->stx_ctime.tv_sec);
-    if ((buf->stx_mask & STATX_BTIME) != 0) {
-        //  Birth time was filled in so use it
-        (*env)->SetLongField(env, attrs, attrs_st_birthtime_sec,
-                             (jlong)buf->stx_btime.tv_sec);
-        (*env)->SetLongField(env, attrs, attrs_st_birthtime_nsec,
-                             (jlong)buf->stx_btime.tv_nsec);
-    } else {
-        //  Birth time was not filled in: fall back to last modification time
-        (*env)->SetLongField(env, attrs, attrs_st_birthtime_sec,
-                             (jlong)buf->stx_mtime.tv_sec);
-        (*env)->SetLongField(env, attrs, attrs_st_birthtime_nsec,
-                             (jlong)buf->stx_mtime.tv_nsec);
-    }
+    (*env)->SetLongField(env, attrs, attrs_st_birthtime_sec, (jlong)buf->stx_btime.tv_sec);
+    (*env)->SetLongField(env, attrs, attrs_st_birthtime_nsec, (jlong)buf->stx_btime.tv_nsec);
+
```

Comment:
I think I need a reminder as to why we put values into these two fields when not available.

Reply:
> I think I need a reminder as to why we put values into these two fields when not available.

There's no reason to do so except to avoid testing availability.

Reply:
It still must test, in order to set the right value for the new available field.

Reply:
I'll change it.